modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/actions/components/ActionForm.tsx

```diff
@@ -157,11 +159,31 @@ const ActionForm = ({
 		method: initialValues?.method ?? '',
 		modalSize: initialValues?.modalSize ?? '',
 		permissionKey: initialValues?.permissionKey ?? '',
+		requestBody: initialValues?.requestBody ?? '',
 		title: initialValues?.title ?? '',
 		type: initialValues?.type ?? 'link',
 		url: initialValues?.url ?? '',
 	} as IAction);
 
+	const isValidJSON = (value: string | undefined) => {
+		if (!value) {
+			return true;
+		}
+
+		if (!value.match(/{[^}]*}/)) {
+			return false;
+		}
+
+		try {
+			JSON.parse(value);
+
+			return true;
+		}
+		catch {
+			return false;
+		}
+	};
```

Comment:
Yes, I agree to renaming utility to `isRequestBodyInputValid`. We are checking here that the value works as a valid request body for the action to use. 

So, an empty value is valid (because the field is not required), but a value without brackets doesn't. That's the reason of the second `if`. For example, `12345` is a valid JSON according to specification, so doesn't trigger a parse error, but isn't valid for our purposes.